examples/cce/create_cluster_node.py

```diff
@@ -20,49 +20,67 @@
 
 
 attrs = {
-    "kind": "Node",
-    "apiVersion": "v3",
-    "metadata": {
-        "name": "myhost",
-        "labels": {
-            "foo": "bar"
+    'kind': 'Node',
+    'apiVersion': 'v3',
+    'metadata': {
+        'name': 'myhost',
+        'labels': {
+            'foo': 'bar'
         },
-        "annotations": {
-            "annotation1": "abc"
+        'annotations': {
+            'annotation1': 'abc'
         }
     },
-    "spec": {
-        "flavor": "s2.large.2",
-        "az": "eu-de-02",
-        "login": {
-            "sshKey": "keypair-pub"
+    'spec': {
+        'flavor': 's2.large.2',
+        'az': 'eu-de-02',
+        'login': {
+            'sshKey': 'keypair-pub'
         },
-        "rootVolume": {
-            "size": 40,
-            "volumetype": "SATA"
+        'rootVolume': {
+            'size': 40,
+            'volumetype': 'SATA'
         },
-        "dataVolumes": [
+        'dataVolumes': [
             {
-                "size": 100,
-                "volumetype": "SATA"
+                'size': 100,
+                'volumetype': 'SATA'
             }
         ],
-        "userTags": [
+        'userTags': [
             {
-                "key": "tag1",
-                "value": "aaaa"
+                'key': 'tag1',
+                'value': 'aaaa'
             },
             {
-                "key": "tag2",
-                "value": "bbbb"
+                'key': 'tag2',
+                'value': 'bbbb'
             }
         ],
-        "k8sTags": {
-            "label-test": "test"
+        'k8sTags': {
+            'label-test': 'test'
+        },
+        'publicIP': {
+            # ids: ['1234', '5678']
+            'count': 2,
+            'eip': {
+                'iptype': '5_bgp',
+                'bandwidth': {
+                    'chargemode': 'traffic',
+                    'size': 10,
+                    'sharetype': 'PER'
+                }
+            }
+        },
+        'count': 2,
```

Comment:
why did you change from `1` to `2`?

Reply:
There are also 2 public IP's in this example. So I wanted to highlight that also 2 nodes can be requested.